src/js/game/LevelMVC/LevelView.js

```diff
@@ -1691,6 +1706,9 @@ module.exports = class LevelView {
       case "treeSpruce": //0,8
         buildTree(this, [0, 8]);
         break;
+      case "treeSpruceSnowy": //1,9
+        buildTree(this, [1, 9]);
```

Comment:
why starting from 1?

Reply:
That was just the way the the json was named. For some reason this one came to me 1-9, not 0-8. That should be changeable if we cared, so long as it lined up. 🤷‍♂️